slack_reporter_config.example.py

```diff
@@ -0,0 +1,4 @@
+auth_token = "xoxb-secret-goes-here" # Should start with "xoxb-"
+board_name = "example board"
+
+channel_id = "C12345ABCDE" # The channel ID, *not* the channel name!
```

Comment:
maybe some instruction as to how to find the channel id? I can see people getting a bit confused and it would be nice for them not to have to turn to a search engine for this